orc8r/cloud/go/services/orchestrator/obsidian/models/conversion.go

```diff
@@ -239,10 +246,47 @@ func (m *MagmadGateway) FromBackendModels(ent configurator.NetworkEntity, device
 	if err == nil {
 		m.Tier = TierID(tierTK.Key)
 	}
+	err = PopulateRegistrationInfo(context.Background(), m, ent.NetworkID)
+	// ignore err and continue returning the rest of the gateway
+	glog.V(2).Infof("failed to populate registration info because %v+", err)
```

Comment:
Shouldn't this log only if `err` is nil?

Reply:
yes nice catch ty